docs/release-notes.md

```diff
@@ -9,23 +9,23 @@
     ![DietPi-Software Bazarr](assets/images/dietpi-software_bazarr.jpg)
     For more details on installation and configuration open [DietPi Optimised Software - Bazarr](../software/bittorrent#bazarr) page.
 
-:    Companion application to Sonarr and Radarr, which manages and downloads subtitles based on your requirements, now available for install. Open [Bazarr](../software/bittorrent#bazarr) page in [Optimised software](../software/).
+ Companion application to Sonarr and Radarr, which manages and downloads subtitles based on your requirements, now available for install. Open [Bazarr](../software/bittorrent#bazarr) page in [Optimised software](../software/).
```

Comment:
There are now sometimes ones, sometimes two and sometimes three white spaces to keep paragraphs as part of the list elements. I suggest to use two in every case, which is the default single tab indentation and that way the text starts exactly below the text of behind the bullet point:
```
- abc
  def
```

Reply:
The number of the leading white spaces I have chosen depending on what shall be displayed.  
I use the atom editor with its internal preview (via ctrl-shift-m, the first ctrl-shift-m always fails...). I made the experience that atom has a very good Markdown preview (not mkdocs preview!) compared e.g. to haroopad etc.

I did the changes using the atom markdown preview function and also propose to edit the releasenotes in this way.